.husky/scripts/check-coverage.sh

```
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash  
+# Exit on error, unset vars, and pipeline failures, like strict mode in JS
+set -euo pipefail
+
+MIN_COVERAGE=70
+
+check_coverage() {
+    echo "🧪 Checking test coverage..."
+
+    # Get the coverage number from the output of sui move coverage summary
+    local coverage=$(sui move coverage summary | awk '/Move Coverage:/ {print $5}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)